src/Combobox/index.js

```diff
@@ -403,6 +404,7 @@ const Combobox = ({
             {...getMenuProps(layerProps)}
             style={{
               width: triggerBounds?.width || "auto",
+              transform: `${errorText ? `translateY(${layerSide === "top" ? "0px" : "-22px"} )` : "none"}`,
```

Comment:
I realize that this has a smell. I might be overlooking a more conventional way of doing this!

Reply:
This expression isn't very complicated. If we find ourselves doing more work than this in the future, we could do something like:

`getMenuStyle(triggerBounds, errorText, layerSide) // returns {}`

```
style={{
    ...getMenuStyle(triggerBounds, errorText, layerSize),
    ...layerProps.style
}}
```

✅ For now, I don't think any of the logic here is complicated enough to justify adding the indirection